pandas/tests/series/test_api.py

```diff
@@ -21,6 +22,12 @@
 
 class SharedWithSparse(object):
 
+    def _assert_series_equal(self, left, right):
```

Comment:
why don't we instead define this in the Mixin classes? (and add a NotImplemented one here)?